test/test_transforms.py

```diff
@@ -291,3 +291,58 @@ def test_gpt2_bpe_tokenizer_save_load_torchscript(self):
         torch.save(tokenizer.__prepare_scriptable__(), tokenizer_path)
         loaded_tokenizer = torch.load(tokenizer_path)
         self._gpt2_bpe_tokenizer((loaded_tokenizer))
+
+
+class TestCLIPTokenizer(TorchtextTestCase):
+    def _load_tokenizer(self, test_scripting):
+        encoder_json = "clip_encoder.json"
+        bpe_vocab = "clip_vocab.bpe"
```

Comment:
Yes they are different. Since our unit test is more like an e2e test, it is better to rely on prod files which we don't expect to change at the moment.

Reply:
As a follow-up, let's also think about how to expose the prod files for tokenizers to the users. If the user just want to use the pre-trained text processing files from established models (XLM-R, Roberta, CLIP), we might want to think how to best expose it. As of now we do not have a standard way of doing so.